File: namsor/ruby/src/PhoneCodeGeoFeedbackLoopExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::SocialApi.new.phone_code_geo_feedback_loop(
        "Teniola", # first_name
        "Apata", # last_name
        "08186472651", # phone_number
        "", # phone_number_e164
        "NG", # country_iso2
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling SocialApi#phone_code_geo_feedback_loop: #{e}"
end

File: namsor/python/src/PhoneCodeGeoFeedbackLoopExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.SocialApi(api_client).phone_code_geo_feedback_loop(
            first_name="Teniola",
            last_name="Apata",
            phone_number="08186472651",
            phone_number_e164="",
            country_iso2="NG",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling SocialApi#phone_code_geo_feedback_loop: %s\n" % e)
